lldb/source/Symbol/Type.cpp

```diff
@@ -1082,6 +1082,19 @@ bool TypeImpl::GetDescription(lldb_private::Stream &strm,
   return true;
 }
 
+CompilerType TypeImpl::FindNestedType(ConstString name) {
```

Comment:
Should an `empty` name require special treatment? Like we don't support that as a valid query?

Reply:
fwiw, `name.IsValid()` would check for both nullptr and empty string

Reply:
Makes sense.
IIRC it's possible to have multiple anonymous unions, so it's not going to work properly even if we allow it.